reactor-core/src/test/java/reactor/core/publisher/TailCallSubscribeTest.java

```diff
@@ -221,7 +221,7 @@ public void retry() throws Exception {
 				.startsWith(
 						tuple(Thread.class.getName(), "getStackTrace"),
 						tuple(stackCapturingPublisher.getClass().getName(), "subscribe"),
-						tuple(Flux.class.getName(), "subscribe"),
+						tuple(InternalFluxOperator.class.getName(), "subscribe"),
```

Comment:
unrelated change? maybe add a `[polish]` commit and tag as `for/merge-with-rebase`?

Reply:
https://github.com/reactor/reactor-core/pull/1884#discussion_r321857934